docs/source/enable_cc_lifecycle.md

```diff
@@ -0,0 +1,300 @@
+  organization.
+* `ORDERER_CA`: the absolute path to the root CA certificate of the organization
+  proposing the system channel update.
+* `ORDERING_NODE_ADDRESS`: the url of an ordering node.
+  When targeting the ordering service, you can target any particular node in the
+  ordering service. Your requests will be forwarded to the leader automatically.
+
+Once you have set the environment variables, navigate to
+[Step 1: Pull and translate the config](./config_update.html#Step-:-Pull-and-translate-the-config).
+
+Once you have a `modified_config.json`, add the lifecycle organization policy
+(as listed in `enable_lifecycle.json`) using this command:
+
+```
+jq -s ".[0] * {\"channel_group\":{\"groups\":{\"Application\": {\"groups\": {\"$ORGNAME\": {\"policies\": .[1].${ORGNAME}Policies}}}}}}" config.json ./enable_lifecycle.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#Step-3-Re-encode-and-submit-the-config).
+
+If every peer organization performs their own channel edit, they can
+edit the configuration without needing further signatures (by default, the only
+signature needed to edit parameters within an organization is an admin of that
+organization). If a different organization proposes the update, then the
+organization being edited will need to sign the channel update request.
+
+#### Edit the application channels
+
+After all of the application channels have been [updated to include V2_0 capabilities](./upgrade_to_newest_version.html#Capabilities),
+endorsement policies for the new chaincode lifecycle must be added to each
+channel.
+
+You can set the same environment you set when updating the peer organizations.
+Note that in this case you will not be updating the configuration of an org in
+the configuration, so the `ORGNAME` variable will not be used.
+
+Once you have set the environment variables, navigate to
+[Step 1: Pull and translate the config](./config_update.html#Step-:-Pull-and-translate-the-config).
+
+Once you have a `modified_config.json`, add the channel endorsement policy
+(as listed in `enable_lifecycle.json`) using this command:
+
+```
+jq -s '.[0] * {"channel_group":{"groups":{"Application": {"policies": .[1].appPolicies}}}}' config.json ./enable_lifecycle.json > modified_config.json
+```
+
+Then, follow the steps at [Step 3: Re-encode and submit the config](./config_update.html#Step-3-Re-encode-and-submit-the-config).
+
+For this channel update to be approved, the policy for modifying the `Channel/Application`
+section of the configuration must be satisfied. By default, this is a `MAJORITY`
+of the peer organizations on the channel.
+
+## Edit channel ACLs (optional)
+
+You can choose to overwrite the default [Access Control List (ACL)](./access_control.html)
+for these new channel endorsement policies by performing a channel update.
+
+If you look at `enable_lifecycle.json`, we can see the fields being added:
```

Comment:
The acls in `enable_lifecycle.json` are the default values. It is better to point it out - e.g., `The following ACLs in enable_lifecycle.json are the default values for the new lifecycle.`

Reply:
Should be done now.